### PR TITLE
Issue 2862 - Market History Fixes

### LINKED ISSUE #2862: [2] No scroll reset on Exchange
**Describe the bug**
Scrolling down a list on the Exchange and switching to a different pair does not reset the scroll position. 

**To Reproduce**
1. Go to Exchange bitCNY/BTS
2. Scroll down MARKET TRADES list till bottom
3. Select a different market RUDEX.WLS/BTS 
4. List looks empty, scrolling is needed to see entries

**Expected behavior**
Show the list from the top

**Screenshots**
empty-lists

**Desktop (please complete the following information):**
 - OS: MX-Linux
 - Version 3.1.190618

### PR DESCRIPTION
**Pull request by startailcoon**
Various MarketHistory fixes (#2862)
- Fix PS when changing market
- Fix inaccurate total
- Code/View split

Tooltip issue code cleaning (#2916)

#### 2862
- 0.75 hrs for fixing the PS issue on market change for the original issue
- 1.25 hrs for code/view splitting

#### 2916
- 0.5 hr for finding the cause of the issue
- 0.5 hr for code cleaning and fixing the issue

Closes #2862, #2916

### CODE REVIEW

**Comment by sschiessl-bcp** [comment]
please resolve conflict

File: app/components/Exchange/MarketHistory.jsx [hunk 1]
```diff
@@ -67,43 +41,90 @@ class MarketHistory extends React.Component {
...
+            }
+        }
+    }
+
+    /***
+     * Update PS Container
+     * type:int [0:destroy, 1:init, 2:update, 3:update w/ scrollTop] (default: 2)
+     */
+    updateContainer(type = 2) {
+        let containerNode = this.refs.view.refs.history;
```
[the remark this answers is not in the document]
Comment (startailcoon) [reply app/components/Exchange/MarketHistory.jsx]: It's the scrollbars we use for everything. I guess it's good for scrolling.

image

[the remark this answers is not in the document]
Comment (sschiessl-bcp) [reply app/components/Exchange/MarketHistory.jsx]: aka you also don't know really? :P

[the remark this answers is not in the document]
Comment (startailcoon) [reply app/components/Exchange/MarketHistory.jsx]: It's a "better" scrollbar implementation than traditional ones, customizable and programmatically scrollable. If it's the best implementation to have among other options, I'm not sure. But this is what it does and what we have :)

**Comment by sschiessl-bcp** [comment]
You need to write 
```
Closes #2862
Closes #2916
```
in order for github to recognize.

**Comment by startailcoon** [comment]
> You need to write
> 
> ```
> Closes #2862
> Closes #2916
> ```
> 
> in order for github to recognize.

You would think that they had that working :bug: :smiling_imp:

**Pull request merged**